mkosi.profiles/obs/fedora.repo/system_systemd.repo

```diff
@@ -0,0 +1,7 @@
+[system_systemd]
```

Comment:
Please name the repo `systemd` instead of `system_systemd` and name the file fedora.repo.